cmd/importer/job.go

```diff
@@ -66,7 +66,6 @@ func doJob(table *table, db *sql.DB, batch int, jobChan chan struct{}, doneChan
 
 	if count > 0 {
 		doInsert(table, db, count)
-		count = 0
```

Comment:
Why remove this line?

Reply:
https://goreportcard.com/report/github.com/pingcap/tidb#ineffassign

Line 69: warning: count assigned and not used (ineffassign)